### PR TITLE
Splash styles

### PR DESCRIPTION
**Pull request by tomjw64**

### CODE REVIEW

File: styles/designs/carnival/parts/_chapterintro-components.scss [hunk 1]
```diff
@@ -86,4 +86,88 @@ $ChapterIntroFirstPara: (
...
+    )
+);
+
+$SplashCaption: (
+    _name: "SplashCaption",
+    _subselector: ' > .os-caption-container',
+    _properties: (
+        position: relative,
+        padding-top: 1rem,
+        padding-bottom: 2rem,
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_chapterintro-components.scss]: Should this use the vertical-rhythm scale?

File: styles/designs/carnival/parts/_chapterintro-components.scss [hunk 2]
```diff
@@ -86,4 +86,88 @@ $ChapterIntroFirstPara: (
...
+        $Caption
+    )
+);
+
+
+$ChapterSplashFigureWrapper: (
+    _name: "SplashFigureWrapper",
+    _subselector: ' > .os-figure.has-splash',
+    _properties: (
+        margin-bottom: 2rem,
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_chapterintro-components.scss]: Should this use the vertical-rhythm scale?

File: styles/designs/carnival/parts/_figureimage-components.scss [hunk 3]
```diff
@@ -5,18 +5,25 @@ $FigureContainer:(
     )
 );
 
+$CaptionWrapper: (
```
Comment (helenemccarron) [inline styles/designs/carnival/parts/_figureimage-components.scss]: 👍

File: styles/designs/carnival/parts/_chapter-shapes.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@import 'chapter-components';
+
+@include create_shape('carnival:::Chapter', (
+  _components: (
+    $chapterContainer,
+  )
+));
```
Comment (tomjw64) [inline styles/designs/carnival/parts/_chapter-shapes.scss]:
```suggestion
));

```

**Pull request merged**